lib/ansible/modules/cloud/docker/docker_container.py

```diff
@@ -692,6 +712,7 @@
     description:
       - Mount a tmpfs directory
     type: list
+    elements: str
```

Comment:
There are some points which could be improved too while you are here :), e.g. 
1. some descriptions end with dots, some not
2. in ```comparisons```'s parameter there are big bunches of texts, could be split
3. some sentences of descriptions are in quotes (without need)
4. too long sentences happen
5. seealso section could be added (at list with general links to the docker documentation, and other related docker modules)

Reply:
These are things that can also be backported to stable-2.8, so I'd rather do that in another PR.